pages/project-version/read/index.js

```diff
@@ -28,7 +28,7 @@ module.exports = settings => {
     res.locals.static.basename = req.fullApplication ? req.buildRoute('projectVersion.fullApplication') : req.buildRoute('projectVersion');
     res.locals.static.projectUrl = req.buildRoute('project.read');
     res.locals.static.establishment = req.project.establishment;
-    res.locals.static.isActionable = req.user.profile.asruUser && get(task, 'data.data.version') === req.versionId;
+    res.locals.static.isActionable = get(task, 'data.data.version') === req.versionId;
```

Comment:
This means admins get a "Nest steps" CTA on project versions when reviewing them for endorsement and not just the inspectors.